mobilecoind/src/payments.rs

```diff
@@ -259,11 +261,17 @@ impl<T: BlockchainConnection + UserTxConnection + 'static, FPR: FogPubkeyResolve
         };
         log::trace!(logger, "Tombstone block set to {}", tombstone_block);
 
+        // Come up with a block version
+        let block_version =
+            BlockVersion::try_from(max(self.ledger_db.get_latest_block()?.version, 1))
```

Comment:
Can you explain (in a comment) why this is needed?

Reply:
I guess we could get rid of it if we make the `TransactionBuilder` support using `BlockVersion(0)`, and it just treats it like block version 1. That might be fine